chichiwang/chichiwang.dev
# Remove boilerplate pages: Home, Documentation

## Description
Linked to Issue: #3
Remove page components `Home` and `Documentation` that were used by the [Tamsui boilerplate](https://github.com/chichiwang/tamsui).

## Changes
* Remove link to `/documentation` page from `app/Layout/Nav.tsx`
* Remove route `/documentation` from `app/dataRoutes/index.ts`
* Remove `/pages/Documentation/` directory (Documentation page)
* Remove `/pages/Home/` directory (Home page)

## Steps to QA
* Pull down and switch to this branch
* Run the app and ensure no issues
* Verify in browser that there are no errors
